src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
 // write code here
+const listEmployee = document.body.querySelectorAll('li');
+
+function parseSalary(salaryString) {
+  return parseFloat(salaryString.replace(/[^0-9.-]+/g, ''));
+}
```

Comment:
The `parseSalary` function is not safe. It uses `parseFloat` on a string that may not be a valid number. This can lead to unexpected results. It would be better to check if the string is a valid number before parsing it.